core/block_svg.js

```diff
@@ -523,14 +524,21 @@ Blockly.BlockSvg.prototype.tab = function(start, forward) {
 
 /**
  * Handle a mouse-down on an SVG block.
- * @param {!Event} e Mouse down event.
+ * @param {!Event} e Mouse down event or touch start event.
  * @private
  */
 Blockly.BlockSvg.prototype.onMouseDown_ = function(e) {
   if (this.workspace.options.readOnly) {
     return;
   }
   if (this.isInFlyout) {
+    // longStart's simulation of right-clicks for longpresses on touch devices
```

Comment:
Updated wording:
    // longStart's simulation of right-clicks for longpresses on touch devices
    // calls the onMouseDown_ function defined on the prototype of the object
    // the was longpressed (in this case, a Blockly.BlockSvg).  In this case
    // that behaviour is wrong, because Blockly.Flyout.prototype.blockMouseDown
    // should be called for a mousedown on a block in the flyout, which blocks
    // execution of the block's onMouseDown_ function.

Still clumsy :/